tests/foreman/api/test_activationkey.py

```diff
@@ -126,6 +130,20 @@ def test_negative_create_3(self):
                         unlimited_content_hosts=True,
                     ).create()
 
+    def test_negative_create_4(self):
```

Comment:
@elyezer I can rename this but then I'd need to rename all other methods. I **can** do this for sure, and I **will** do it, but I'd recommend that such feedback be added to a generic task so that we all start doing it too.